src/graph/transform/kdtree_ndarray_adaptor.h

```diff
@@ -0,0 +1,115 @@
+/*!
+ *  Copyright (c) 2021 by Contributors
+ * \file graph/transform/kdtree_ndarray_adaptor.h
+ * \brief NDArray adaptor for nanoflann, without
```

Comment:
`adapter` - this spelling is more common in our codebase.
Please also scan for other occurrences (including class names such as `KDTreeNDArrayAdapter`.